README.md

```
@@ -8,16 +8,113 @@
 
+As a minimum, a panel bed file and binned coverage data must be provided as input. The panel bed file is expected to have the following columns (with no header):
 
-## <img src="athena/data/images/moby.png" width="34"/> Docker
+* chrom (`str`)
+* start (`int`)
+* end (`int`)
+* gene (`str`)
+* transcript (`str`)
+* region (`str`)
 
+The coverage data is expected to be binned data as output from a tool such as (mosdepth)[mosdepth-url], with the columns:
```

Markdownlint (0.37.0)

(mosdepth)[mosdepth-url]
Reversed link syntax

(MD011, no-reversed-links)